lib/magma/commands.rb

```diff
@@ -2,6 +2,38 @@
 require 'logger'
 
 class Magma
+  class LoadProject < Etna::Command
+    usage '[project_name, path/to/file.json] # Import attributes into database for given project name from JSON file'
+
+    def options
+      Magma::Attribute.options - [:loader] + [:created_at, :updated_at, :attribute_name]
+    end
+
+    def execute(project_name, file_name)
+      file = File.open(file_name)
+      file_data = JSON.parse(file.read, symbolize_names: true)
+
+      db = Magma.instance.db
+      models = file_data[:models]
+
+      models.keys.each do |model|
+        model_name = models[model][:template][:name]
+        attributes = models[model][:template][:attributes]
+        attributes.values.each do |attribute|
+          attribute_type = attribute[:attribute_type]
+          attribute.slice!(*options)
+          attribute.merge!(
+            project_name: project_name, 
```

Comment:
We don't do any validation of the input JSON in this command, so we must assume things are well-formed. Perhaps this should change at some point, but at this point we might want to at least ensure that `attribute[:attribute_name]` exists by putting it in the merge - we should be able to guarantee it if we iterate as `attributes.each do |attribute_name, attribute|`

Reply:
👍🏼